Podfile

```diff
@@ -2,7 +2,13 @@ platform :ios, '9.0'
 use_frameworks!
 
 def shared_pods
-    pod 'Mapbox-iOS-SDK', '~> 4.8.0'
+    # This uses a local build of the Maps SDK if you are installing from within mapbox-gl-native
+    if Dir.pwd.include?('mapbox-gl-native')
+      pod 'Mapbox-iOS-SDK', :path => '../../../build/ios/pkg/dynamic/Mapbox-iOS-SDK.podspec'
```

Comment:
We should note that this file/build won’t exist unless the developer does a `make iframework`.